src/legacy/ui/public/vis/editors/default/vis_options.js

```diff
@@ -37,8 +37,10 @@ uiModules
     ['uiState', { watchDepth: 'collection' }],
     ['setValue', { watchDepth: 'reference' }],
     ['setValidity', { watchDepth: 'reference' }],
+    ['setVisType', { watchDepth: 'reference' }],
     ['setTouched', { watchDepth: 'reference' }],
     'hasHistogramAgg',
+    'currentTab',
```

Comment:
Do you use it anywhere?

Reply:
'currentTab' isn't used. It needs to watch the tab selection changes and to rerender react component. There is an issue when a user switches tabs, EuiAccordion calculates wrong as 0 px. It occurs because when a tab isn't selected it has display: none and EuiAccordion calculates its height as 0px since its content is hidden. When that tab is selected, EuiAccordion doesn't recalculate the height because react rendering wasn't triggered (none from the props weren't updated).
To rerender React component I added currentTab param to watch in vis-editor-vis-options and vis-editor-agg-group directives. So now when a tab is changed, EuiAccordion counts its height again.